k4GaudiPandora/src/DDCaloHitCreator.cc

```diff
@@ -44,25 +39,27 @@
 //forward declarations. See in DDPandoraPFANewProcessor.cc
 
 // dd4hep::rec::LayeredCalorimeterData * getExtension(std::string detectorName);
-dd4hep::rec::LayeredCalorimeterData* getExtension(unsigned int includeFlag, unsigned int excludeFlag = 0);
+dd4hep::rec::LayeredCalorimeterData* getExtension(unsigned int includeFlag, MsgStream log, unsigned int excludeFlag = 0);
```

Comment:
```suggestion
dd4hep::rec::LayeredCalorimeterData* getExtension(unsigned int includeFlag, MsgStream& log, unsigned int excludeFlag = 0);
```

I don't think we want a copy of the MsgStream here.

edit: Thinking about it a bit more: I think we actually don't want to use the iLCSoft facilities for logging, but rather go through the ones Gaudi provides.

Reply:
Isn't the MsgStream the Gaudi logging service? This was my quick and dirty way of giving this function access to the proper log, but if there's a better way, I'll take it!